### PR TITLE
State viewport height assumption and make a test fit that definition

### PR DESCRIPTION
**Pull request by annevk**
This matches the changes made to the img element test in #7644.

### CODE REVIEW

File: docs/writing-tests/assumptions.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The tests make a number of assumptions of the user agent, and new
 tests can freely rely on these assumptions being true:
 
  * The device is a full-color device.
- * The device has a viewport width of at least 800px.
+ * The device has viewport dimensions of at least 800px by 600px.
```
Comment (Ms2ger) [inline docs/writing-tests/assumptions.md:7]: Is this RFC material?

Comment (annevk) [reply docs/writing-tests/assumptions.md:7]: I wouldn't know. I was hoping that given the discussion in #7642 this was just missing information. Not sure I want to make the time to write an RFC for this.

Comment (jgraham) [reply docs/writing-tests/assumptions.md:7]: I think it's probably OK? Although having said that I don't think we currently enforce a minimum window size in wptrunner, except for reftests, so maybe we should have a followup issue to actually implement this? In which case a small RFC wouldn't hurt I think.

Comment (annevk) [reply docs/writing-tests/assumptions.md:7]: I filed https://github.com/web-platform-tests/rfcs/issues/188 as follow-up. Is that sufficient to land this change? I primarily care about getting this test fixed, but it seems good to update the documentation as well as per the discussion many years ago.

**Review by jgraham — APPROVED** [review]

**Review by Smieleski — COMMENTED** [review]
html/semantics/embedded-content/the-object-element/usemap-casing.html

**Pull request merged**